backends/qualcomm/scripts/build.sh

```diff
@@ -71,6 +71,7 @@ if [ "$BUILD_AARCH64" = true ]; then
         -DCMAKE_INSTALL_PREFIX=$BUILD_ROOT \
         -DEXECUTORCH_BUILD_QNN=ON \
         -DEXECUTORCH_BUILD_SDK=ON \
+        -DEXECUTORCH_BUILD_EXTENSION_MODULE=ON \
```

Comment:
Same here, a seperate PR

Reply:
This is related to `qnn_llama_runner`, I think we still need to land it together.

Reply:
Our custom runner ('llama2/runner/runner') invokes methods inside `extension/module/module.cpp`. We could get rid of it if required.